pennylane/templates/layers/basic_entangler.py

```diff
@@ -120,6 +120,37 @@ def circuit(weights):
 
         Accidentally using a gate that expects more parameters throws a
         ``ValueError: Wrong number of parameters``.
+
+        !!! Update for more than one single-qubit rotation gates !!!
+        The basic entangler can now take more than one gate as parameter for repetition. The code works as follows: For a given list of operations, rotations = [RX,RY,RZ], the model takes in parameters in shape (num_layers,num_wires*num_rotations)
+         where num_rotations = len(rotations). Meaning that each rotation takes a parameter per layer per wire.
+        The function compute decomposition checks the form of rotations, if it is a single roation element the code works as it is used to.
+        For a list, it adds the parameters in order.
+        Example:
+        For a two layer, 3 rotations and 4 wires system, the params have the form
```

Comment:
Make sure that you are following ReST (restructedtext) conventions for doc-strings (similar to markdown, which is used for the changelog).

You can always check how the docs look by looking at the github actions 
![image](https://user-images.githubusercontent.com/43949391/179541196-8884bd12-e0ce-4d61-8bce-df5a4182cc67.png)
or by building the docs locally with `make docs` on your machine. See https://pennylane.readthedocs.io/en/stable/development/guide/documentation.html and